reana/reana_bench/config.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please remove all coding declarations. We don't use them elsewhere, and I think they are nont really necessary anymore, since [Python documentation](https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations) says _"If no encoding declaration is found, the default encoding is UTF-8."_

Reply:
I was following files from reana-dev. I will remove the coding declaration from benchmark files.

Reply:
Done.

Reply:
Already removed it from `reana-benchmark`. I think it is a good first move. We can think about removing declarations in other places later.